src/config/web3/utils/convert.ts

```diff
@@ -0,0 +1,51 @@
+import Web3 from 'web3';
+import { u8aToHex } from '@polkadot/util';
+import { addressToEvm, evmToAddress } from '@polkadot/util-crypto';
+import { isValidAddressPolkadotAddress } from 'src/hooks/helper/plasmUtils';
+import { ASTAR_SS58_FORMAT } from 'src/hooks/helper/plasmUtils';
+
+// Memo: The EVM address won't be same as the address shown in MetaMask imported from the same private key of the SS58
+// Ref: https://github.com/polkadot-js/common/issues/931
+export const toEvmAddress = (ss58Address: string) => {
+  return u8aToHex(addressToEvm(ss58Address));
+};
+
+export const checkSumEvmAddress = (evmAddress: string): string => {
+  const web3 = new Web3();
```

Comment:
The Web3 object is being instantiated multiple times just to call the utils function. I recommend using a different library for doing this.

Reply:
@sirius651 @hoonsubin 
How about use [ethers.utils. getAddress](https://docs.ethers.io/v5/api/utils/address/#utils-getAddress) instead? `ethers` doesn't required for creating the new instance here.

Reply:
@sirius651 @hoonsubin
How about use [ethers.utils.getAddress](https://docs.ethers.io/v5/api/utils/address/#utils-getAddress) instead? `ethers` doesn't required for creating the new instance here.

```suggestion
export const checkSumEvmAddress = (evmAddress: string): string => {
  return ethers.utils. getAddress(evmAddress)
}
```

Reply:
thanks. fixed it based on your suggestion. but I'm curious isn't it better to only use `either` or `web3` instead of both?

Reply:
That's a good point. I like to use some utils that come with `ethers.js`, but more familiar to use `web3.js` whenever working with `wallet provider`.

Reply:
okay, that would be another task atm.